pkg/pillar/cmd/zedrouter/pubsubhandlers.go

```diff
@@ -29,6 +29,17 @@ func (z *zedrouter) handleGlobalConfigModify(ctxArg interface{}, key string,
 	z.handleGlobalConfigImpl(ctxArg, key, statusArg)
 }
 
+func (z *zedrouter) configureGOGC(gcp *types.ConfigItemValueMap) {
```

Comment:
I don't think that zedrouter is the correct place for this.
Zedrouter is responsible for managing network instances.
Better place would be for example zedmanager.

Reply:
Right, missed.